Gruntfile.coffee

```diff
@@ -117,12 +119,32 @@ module.exports = (grunt) ->
         files:
           "dist/brand-buttons.min.css": "<%= yeoman.app %>/brand-buttons.scss"
           "dist/brand-buttons-inversed.min.css": "<%= yeoman.app %>/brand-buttons-inversed.scss"
+    # Compiles LESS to CSS and generates necessary files if requested
+    less:
+      server:
+        files:
+          ".tmp/brand-buttons.css": "<%= yeoman.app %>/less/brand-buttons.less"
+          ".tmp/brand-buttons-inversed.css": "<%= yeoman.app %>/less/brand-buttons-inversed.less"
+
+      dist:
+        options:
+          sourceComments: "none"
+          sourceMap: true
+
+        files:
+          "dist/lesscss/brand-buttons.css": "<%= yeoman.app %>/less/brand-buttons.less"
+          "dist/lesscss/brand-buttons-inversed.css": "<%= yeoman.app %>/less/brand-buttons-inversed.less"
+      min:
+        options:
+          compress: true
+        files:
+          "dist/lesscss/brand-buttons.min.css": "<%= yeoman.app %>/less/brand-buttons.less"
+          "dist/lesscss/brand-buttons-inversed.min.css": "<%= yeoman.app %>/less/brand-buttons-inversed.less"
 
-    
     # Run some tasks in parallel to speed up the build process
     concurrent:
-      server: ["sass:server"]
-      dist: ["sass:dist", "sass:min"]
+      server: ["sass:server", "less:server"]
+      dist: ["sass:dist", "sass:min", "less:dist", "less:min"]
```

Comment:
Pick one. `/dist/brand-buttons.css` always be the same (from Sass or Less), so why we need run twice?